applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c

```diff
@@ -1252,13 +1253,34 @@ static void unicast_discovery_complete_cb(struct bt_conn *conn, int err,
 					  const struct bt_csip_set_coordinator_set_member *member,
 					  const struct bt_csip_set_coordinator_csis_inst *csis_inst)
 {
-	if (err) {
+	int ret;
+	struct le_audio_msg msg;
+	uint8_t index;
+
+	ret = device_index_get(conn, &index);
+	if (ret) {
+		return;
```

Comment:
Add a print?

Reply:
There is already a print in the device_index_get